client/app/bundles/Zoon/components/WorkflowCreator.jsx

```diff
@@ -187,6 +188,21 @@ class WorkflowCreator extends React.Component {
     return this.families().length
   }
 
+  showModuleLightbox = (moduleId) => {
+    if (moduleId === null || moduleId === undefined) {
```

Comment:
What triggered my comment is the fact that there are two different checks (`null` and `undefined`), and we're only using `null`.

> The internet is against that in general, but I'm pretty ok with the behavior here.
Care to share some sources? I'm ok with leaving it as is, but I'd like to read a bit about it.